Formula/monero.rb

```diff
@@ -1,8 +1,9 @@
 class Monero < Formula
-  desc "Official monero wallet and cpu miner"
+  desc "Secure, private, untraceable cryptocurrency"
   homepage "https://getmonero.org/"
-  url "https://github.com/monero-project/monero/archive/v0.11.1.0.tar.gz"
-  sha256 "b5b48d3e5317c599e1499278580e9a6ba3afc3536f4064fcf7b20840066a509b"
+  url "https://github.com/monero-project/monero.git",
+      :tag => "v0.12.0.0",
```

Comment:
Please use the [tarball URL](https://github.com/monero-project/monero/archive/v0.12.0.0.tar.gz) here, and not a `git` tag/commit.

Reply:
There is git submodule and tarball is missing it.